neofetch

```diff
@@ -274,6 +279,10 @@ get_model() {
         "Solaris")
             model="$(prtconf -b | awk -F':' '/banner-name/ {printf $2}')"
         ;;
+
+        "AIX")
+            model="$(prtconf | awk -F':' '/System Model/ {printf $2}')"
```

Comment:
Maybe we can merge this and Solaris?

`awk '/banner-name|System Model/ {printf $2}'`